### PR TITLE
[API] Adding new fields to project/images endpoint

### PR DESCRIPTION
**Pull request by xlecours**
This add new fields to project/images API endpoint

- PSCID
- Visit_date
- Site
- QC_status
- Selected

It should at least provide a way to filter data once the result is send to the user.

A further step will be to add filter option server side.

### CODE REVIEW

File: docs/API/LorisRESTAPI_v0.0.3.md [hunk 1]
```diff
@@ -153,8 +153,13 @@ Will return a JSON object of the form:
   "Images" : [
     {
       "Candidate": "123456",
+      "PSCID": "MTL001",
       "Visit": "V1",
+      "Visit_date": "2016-08-09",
```
Comment (driusan) [inline docs/API/LorisRESTAPI_v0.0.3.md]: This is ambiguous and should be described in the text. Which date is it? The date from the scan header? The date from the session? The date from the mri_parameter_form? Etc...

File: docs/API/LorisRESTAPI_v0.0.3.md [hunk 2]
```diff
@@ -153,8 +153,13 @@ Will return a JSON object of the form:
...
+      "Site": "Montreal Neurological Institute",
       "ScanType": "t2", /* Acquisition protocol */
+      "QC_status": "Pass|Fail",
+      "Selected": "true|false",
```
Comment (MounaSafiHarab) [inline docs/API/LorisRESTAPI_v0.0.3.md]: There is a third option which is NULL, shouldn't it be listed?

File: docs/API/LorisRESTAPI_v0.0.3.md (hunk 1 above)
Comment (MounaSafiHarab) [reply docs/API/LorisRESTAPI_v0.0.3.md]: this Visit date would be NULL for phantoms (always) and for candidates whose MRI visit has not started yet; is it worth specifying this?

**Comment by MounaSafiHarab** [comment]
@xlecours 

see my comments/screenshots. I just had a comment on the NULL option in the QC status, Visit_date and Selected; otherwise, works as you describe.

**Comment by driusan** [comment]
Not sure if null should be in quotes or if the field should just be excluded in that case, but it's not really a blocker

**Pull request merged**